snapshot/archive.go

```diff
@@ -17,13 +17,13 @@ type ArchiveFormat = string
 
 const (
 	ArchiveTar     ArchiveFormat = "tar"
-	ArchiveTarball               = "tarball"
-	ArchiveZip                   = "zip"
+	ArchiveTarball ArchiveFormat = "tarball"
+	ArchiveZip     ArchiveFormat = "zip"
 )
```

Comment:
also, fwiw, I *personally* don't like this; I'd prefer the impicit type rather than spell it everytime.  Not complaining this time, just raising my opinion to hear from what other think.

Reply:
Resolving this comment, see https://github.com/PlakarKorp/plakar/pull/317#issuecomment-2619062380

I personally don't mind about this syntax, but I also don't mind if we prefer to update the linter's configuration.